packages/fluentui/docs/src/examples/components/Checkbox/BestPractices/CheckboxBestPractices.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+import ComponentBestPractices from '../../../../components/ComponentBestPractices';
+
+const doList = ['Add `controlsIds="id1 id2"` for indeterminate checkbox referring to the controls which it depends'];
```

Comment:
https://github.com/microsoft/fluentui/pull/16081/files#r571040798

Reply:
?

Reply:
```suggestion
const doList = ['Add `aria-controls="id1 id2"` for indeterminate checkbox referring to the controls which it depends'];
```

Do we need `controlsIds` or we can just specify `aria-controls`?